test/utils/test_scatter.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import torch
+import torch_scatter
+
+from torch_geometric.utils import scatter
+
+torch.manual_seed(12345)
```

Comment:
If "random" `index` below would lack some number from the requested range, for `mul` algorithm default value in PyTorch would be `0`, while in `torch_scatter` it would be `1` (I don't think that's correct). Setting a fixed seed will prevent sporadical errors.